README.md

```
@@ -14,182 +23,94 @@ flowchart TD
-   This is a complete single-tenant deployment as-is.
-3. Visit `http://localhost:8080/` and create the owner account through the setup wizard.
+2. Open `http://localhost:8080/` and create the owner account through the setup wizard.
+
+3. Generate an API key and start creating entities.
 
-Prefer building from source? Clone the repo, place the model files as in step 1, then `make init` (needs Docker Compose and `make`) builds and starts PostgreSQL plus the app:
+For building from source:
 
 ```console
 $ git clone https://github.com/yorishiro-ai/yorishiro && cd yorishiro
 $ make init
```

markdownlint-cli2 (0.23.2)

[warning] Dollar signs used before commands without showing output

(MD014, commands-show-output)